mapentity/static/mapentity/mapentity.detail.js

```diff
@@ -0,0 +1,8 @@
+$(document).ready(function () {
+    var url_string = window.location.href;
+    var url = new URL(url_string);
+    var tab = url.searchParams.get("tab");
+    if (tab !== null) {
+        $('#tab-' + tab).find('a').click();
```

Comment:
don't use find('a'). the real problem that is ID should be defined on a tag, no on li tag

Reply:
lokk at example : https://getbootstrap.com/docs/4.0/components/navs/#javascript-behavior